alacritty/src/event.rs

```diff
@@ -930,7 +928,6 @@ impl<'a, N: Notify + 'a, T: EventListener> ActionContext<'a, N, T> {
     fn exit_search(&mut self) {
         self.display.pending_update.dirty = true;
         self.search_state.history_index = None;
-        *self.dirty = true;
```

Comment:
This seems **really** wrong. There definitely should be some dirtiness going on when exiting the search. At the very least when `focused_match != None`.

Reply:
There's dirty already `self.display.pending_update.dirty = true;`. No need to set it twice.